aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1562,6 +1562,41 @@ static inline Tensor & sparse_transpose_(Tensor & self, int64_t dim0, int64_t di
   return self;
 }
 
+static inline Tensor sparse_csr_transpose(const Tensor & self, int64_t dim0, int64_t dim1) {
+  TORCH_INTERNAL_ASSERT(self.is_sparse_csr());
+  auto dim0_ = maybe_wrap_dim(dim0, 2);
```

Comment:
These are unused which trips up a test

Reply:
Fixed. In fact, sparse_csr_transpose does not need the dimensions arguments.